.drone.star

```diff
@@ -1451,13 +1451,13 @@ def ocisWebService():
 		'pull': 'always',
 		'detach': True,
 		'environment' : {
-			'PHOENIX_WEB_CONFIG': '/srv/config/drone/config.json',
-			'PHOENIX_ASSET_PATH': '/var/www/owncloud/web/dist',
-			'PHOENIX_OIDC_CLIENT_ID': 'phoenix'
+			'WEB_UI_CONFIG': '/srv/config/drone/config.json',
+			'WEB_ASSET_PATH': '/var/www/owncloud/web/dist',
+			'WEB_OIDC_CLIENT_ID': 'phoenix'
 		},
 		'commands': [
 			'cd /var/www/owncloud',
-			'./ocis-phoenix --log-level debug server',
+			'./ocis-web --log-level debug server',
```

Comment:
Hmmm - I  think this should be running `./web`. `./ocis-web` will be the version of `web` that has been "built-in" to `owncloud/ocis`

Reply:
This has been changed in a recent commit.